ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Header, Segment } from 'semantic-ui-react'
+import styled from 'styled-components'
+
+import { FILE_FIELD_NAME, PROJECT_DESC_FIELD, GENOME_VERSION_FIELD, FAMILY_FIELD_ID, INDIVIDUAL_FIELD_ID } from 'shared/utils/constants'
+import { BaseBulkContent, BASE_UPLOAD_FORMATS } from 'pages/Project/components/edit-families-and-individuals/BulkEditForm'
+import { INDIVIDUAL_CORE_EXPORT_DATA, INDIVIDUAL_ID_EXPORT_DATA } from 'pages/Project/constants'
+import ReduxFormWrapper from 'shared/components/form/ReduxFormWrapper'
+import { BooleanCheckbox } from 'shared/components/form/Inputs'
+import { VerticalSpacer } from 'shared/components/Spacers'
+
+const StyledLabel = styled.label`
+  color: rgba(0,0,0,.87);
+  font-size: .92857143em;
+  font-weight: 700;
+`
+
+const FIELD_DESCRIPTIONS = {
+  [FAMILY_FIELD_ID]: 'Family ID',
+  [INDIVIDUAL_FIELD_ID]: 'Individual ID (needs to match the VCF ids)',
+}
+const REQUIRED_FIELDS = INDIVIDUAL_ID_EXPORT_DATA.map(config => (
+  { ...config, description: FIELD_DESCRIPTIONS[config.field] }))
+
+const BLANK_EXPORT = {
+  filename: 'individuals_template',
+  rawData: [],
+  headers: [...INDIVIDUAL_ID_EXPORT_DATA, ...INDIVIDUAL_CORE_EXPORT_DATA].map(config => config.header),
+  processRow: val => val,
+}
+
+const UploadPedigreeField = React.memo(() =>
+  <div>
+    {/* eslint-disable-next-line jsx-a11y/label-has-for */}
+    <StyledLabel>Upload Pedigree Data</StyledLabel>
```

Comment:
To make the label style be consistent with other fields, I add custom styling to it. Is it okay or not?

Reply:
Labels should render properly inside forms, and adding custom styling to get things to look like default semantic ui usually means we are doing something wrong. My guess is that the issue is the exra nested div here. If you inspect the rendered html, my guess is what you see here is something like 
```
<div class="field">
  <div>
    <label>
```
and semantic ui is expecting 
```
<div class="field">
  <label>
```
The way to fix this here is is to change this component to a list instead of wrapping everything in an extra div. Remmeber with lists you always need to add a `key` prop to each element. So this function would return 
```
[
  <label key="label>Upload Pedigree Data</label>,
  <Segment key="uploadForm">....</Segment>,
]
```